tests/test_transmuters.py

```diff
@@ -1,6 +1,6 @@
 """Transmuter tests"""
 from nose.tools import assert_equal, assert_almost_equal
-
+import nose
```

Comment:
Why do we need to import `nose` here?  We import from `nose.tools` above - did it cause an error?

Reply:
The nose module is used to run the test function `test_transmuters_cram()` as a test suite. The `nose.tools` module contains helper functions that can be used to write tests with `nose`, but `nose` is the actual test runner that executes the test functions and reports the results.

The `if __name__ == "__main__":` block at the end of the script is used to run the test suite if the script is being executed directly, rather than being imported as a module. In this case, `nose.runmodule()` is called to run the `test_transmuters_cram()` function as a test suite using nose.

Reply:
Thanks. We must not test in this way so didn't see it fail in the past.